api/app/resources/bookings/booking/booking_list.py

```diff
@@ -33,7 +33,7 @@ def get(self):
         csr = CSR.find_by_username(g.oidc_token_info['username'])
         office_filter = csr.office_id
 
-        if request.args.get('office_id') and csr.role.role_code == "LIAISON":
+        if request.args.get('office_id') and csr.liaison_designate == 1:
```

Comment:
Actually not all of them. This one is fine.

Might be good to extract a method onto the CSR object for `is_liaison`, `is_pesticide`, etc, would make this much more readable.

Reply:
I'll cover this in a separate PR.

Reply:
Fixed negations tho